server/webapp/localize_ja.xml

```diff
@@ -171,6 +171,7 @@
     <entry key="USER_SUCCESSFULLY_ADDED">User ''{0}'' successfully added.</entry>
     <entry key="USER_ALREADY_EXISTS">Failed to add user. User ''{0}'' already exists.</entry>
     <entry key="USER_DOES_NOT_EXIST_IN_DB">User ''{0}'' does not exist in the database.</entry>
+    <entry key="INVALID_LDAP_ERROR">There was an error authenticating you. Please check the server logs, or contact your the go administrator.</entry>
```

Comment:
Administrators are all knowing super humans :-)
But yeah, what's a better message here? This error happens when an admin makes a mistake in configuring we can drop the contact administrator.

How about
```
Cannot authenticate, this might be a problem with security configuration or setup.
```